src/notation/internal/notationactioncontroller.cpp

```diff
@@ -408,7 +408,7 @@ void NotationActionController::init()
     registerAction("add-down-bow", &Interaction::toggleArticulation, Ms::SymId::stringsDownBow);
     registerAction("transpose-up", &Interaction::transposeSemitone, 1, PlayMode::PlayNote);
     registerAction("transpose-down", &Interaction::transposeSemitone, -1, PlayMode::PlayNote);
-    registerAction("toggle-insert-mode", &Interaction::toggleGlobalOrLocalInsert);
```

Comment:
The `Interaction::toggleGlobalOrLocalInsert` method (`NotationInteraction::toggleGlobalOrLocalInsert`) is now unused. It would be good to remove it.